src/Tribe/Tickets_View.php

```diff
@@ -0,0 +1,516 @@
+			// Ignore RSVP if we don't tell it specifically
+			if ( 'rsvp' === $attendee['provider_slug'] && ! $include_rsvp ) {
+				continue;
+			}
+
+			// If we have a user_id then test it and ignore the ones that don't have it
+			if ( ! is_null( $user_id ) ) {
+				if ( empty( $attendee['user_id'] ) || $attendee['user_id'] != $user_id ) {
+					continue;
+				}
+			}
+
+			$orders[ (int) $attendee['order_id'] ][] = $attendee;
+		}
+
+		return $orders;
+	}
+
+	/**
+	 * Fetches from the Cached attendees list the ones that are relevant for this user and event
+	 * Important to note that this method will bring the attendees from RSVP
+	 *
+	 * @param  int       $event_id     The Event ID it relates to
+	 * @param  int|null  $user_id      An Optional User ID
+	 * @return array                   Array with the RSVP attendees
+	 */
+	public function get_event_rsvp_attendees( $event_id, $user_id = null ) {
+		$all_attendees = Tribe__Tickets__Tickets::get_event_attendees( $event_id );
+		$attendees = array();
+
+		foreach ( $all_attendees as $key => $attendee ) {
+			// Skip Non RSVP
+			if ( 'rsvp' !== $attendee['provider_slug'] ) {
+				continue;
+			}
+
+			// If we have a user_id then test it and ignore the ones that don't have it
+			if ( ! is_null( $user_id ) ) {
+				if ( empty( $attendee['user_id'] ) || $attendee['user_id'] != $user_id ) {
+					continue;
+				}
+			}
+
+			$attendees[] = $attendee;
+		}
+
+		return $attendees;
+	}
+
+	/**
+	 * Gets a List of Possible RSVP answers
+	 * @return array
+	 */
+	public function get_rsvp_options() {
+		$options = array(
+			'yes' => __( 'Going', 'event-tickets' ),
+			'no' => __( 'Not Going', 'event-tickets' ),
+		);
+
+		/**
+		 * Allow users to add more RSVP options
+		 * @param array $options
+		 */
+		return apply_filters( 'tribe_tickets_rsvp_options', $options );
+	}
+
+	/**
+	 * Check if the RSVP is a valid option
+	 *
+	 * @param  string  $option Which rsvp option to check
+	 * @return boolean
+	 */
+	public function is_valid_rsvp_option( $option ) {
+		return in_array( $option, array_keys( $this->get_rsvp_options() ) );
+	}
+
+	/**
+	 * Counts the Amount of RSVP attendees
+	 *
+	 * @param  int       $event_id     The Event ID it relates to
+	 * @param  int|null  $user_id      An Optional User ID
+	 * @return int
+	 */
+	public function count_rsvp_attendees( $event_id, $user_id = null ) {
+		$rsvp_orders = Tribe__Tickets__Tickets_View::get_event_rsvp_attendees( $event_id, $user_id );
+		return count( $rsvp_orders );
+	}
+
+	/**
+	 * Counts the Amount of Tickets attendees
+	 *
+	 * @param  int       $event_id     The Event ID it relates to
+	 * @param  int|null  $user_id      An Optional User ID
+	 * @return int
+	 */
+	public function count_ticket_attendees( $event_id, $user_id = null ) {
+		$ticket_orders = Tribe__Tickets__Tickets_View::get_event_attendees_by_order( $event_id, $user_id );
+		$i = 0;
+		foreach ( $ticket_orders as $orders ) {
+			foreach ( $orders as $attendee ) {
```

Comment:
Might something like `$i += count( $orders )` achieve the same result with less overhead/less code (that an inner foreach)?